tests/unit/ssg/test_ansible.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+import os
+import re
+import ssg.ansible
+
+
+def no_whitespace_compare(left, right):
+    return re.sub(r'\s', '', left) == re.sub(r'\s', '', right)
+
+
+def test_add_minimum_version():
+    good_snippet = """
+ - hosts: all
+   vars:
+   tasks:
+"""
```

Comment:
I'm not sure if we want this indented. I originally de-indented these because of whitespace differences between that and the output, but then I added the `no_whitespace_compare` function and decided to just ignore whitespace altogether. 

However, I find that four space indent on the triple quotes look funny (because the rest have to be 3 spaces because YAML....). Thoughts?